dtoa.opam

```diff
@@ -1,4 +1,5 @@
 opam-version: "2.0"
+synopsis: "Converts OCaml floats into strings, using the efficient Grisu3 algorithm."
```

Comment:
see line 19, but yours is probably better.

Reply:
lol, didn't see it on the end of file